Classes/BEMCheckBox.h

```diff
@@ -8,6 +8,7 @@
 
 #import <UIKit/UIKit.h>
 
+@class BEMCheckBoxGroup;
```

Comment:
Yes, it seems like the general recommendation is to declare everything you need in the header, and then do your actual imports in the implementation file. Yet another issue that Swift rids us of.